### ISSUE TITLE
vue-loader 13 compatible ?

### ISSUE THREAD

**Issue by Blatty** [adeptoas/sweet-modal-vue#28]
After upgrading to Vue 2.4 & vue-loader 13.x, all my sweet-modal components are not working:

```
[Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <SweetModal>
       <Hello> at src/components/Hello.vue
          <App> at src/App.vue
            <Root>
```

```
vue.esm.js?efeb:566 TypeError: this.$refs.mobileMenu.close is not a function
```

Am I the only one getting this error? How to fix it?

**Comment by Novarg** [comment]
Same here, @Blatty do you have any solution?

**Comment by Blatty** [comment]
Thanks @Novarg, that helped & resolved my issue!

**Comment by Novarg** [comment]
@Blatty can you reopen ticket so package author can be aware and make appropriate changes?

**Comment by bluefirex** [comment]
Fixed in 79bf59e80c4d80abb0bab6a92e1a182de9a8ec02, released in [v1.2.0](https://github.com/adeptoas/sweet-modal-vue/releases/tag/v1.2.0)
